### ISSUE TITLE
typo in http://code.highcharts.com/highcharts.src.js line 2873

### ISSUE THREAD

**Issue by daemth** [highcharts/highcharts#4616]
I guess the Â isn't supposed to be there:

![schermafdruk 2015-10-02 10 36 27](https://cloud.githubusercontent.com/assets/3327763/10242497/79f5cc3c-68f1-11e5-9ef0-662d23810b7c.png)

``` javascript
        visibilitySetter: function (value, key, element) {
            // IE9-11 doesn't handle visibilty:inherit well, so we remove the attribute instead (#2881, #3909)
            if (value === 'inherit') {
                element.removeAttribute(key);
            }Â else {
                element.setAttribute(key, value);
            }
        },
```

**Comment by highslide-software** [comment]
Thanks, it's already fixed in 1d2d1e2c6e63cf259395ea5c30730407bcaf0bac

**Comment by zhy1stgg** [comment]
thanks
